spec/models/concerns/wakeable_spec.rb

```diff
@@ -448,5 +448,18 @@ def my_method
         expect(wakeable.wakes_resource).to have_wakes_graph(:canonical_location => '/some-new-title')
       end
     end
+
+    describe 'on destroy' do
+      it 'destroys the wakes resource and wakes locations' do
+        wakeable = model_class.create(:title => 'Some Title')
+        expect(Wakes::Resource.find_by(:label => 'Some Title')).to be_a(Wakes::Resource)
+        expect(Wakes::Location.find_by(:path => '/some-title')).to be_a(Wakes::Location)
+
+        wakeable.destroy
+
+        expect(Wakes::Resource.find_by(:label => 'Some Title')).to be_nil
+        expect(Wakes::Location.find_by(:path => '/some-title')).to be_nil
```

Comment:
@benhutton Just curious, why not save the reference to the `Wakes::Resource` and `Wakes::Location` above and check for them by `id` here? Seems like that would be better assurance then checking by `label`/`path`.